ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml

```diff
@@ -78,7 +78,7 @@ Popup {
             appMain.changeAppSection(Constants.profile)
             // TODO: replace with shared store constant
             // Profile/RootStore.notifications_id
-            profileLayoutContainer.changeProfileSection(7)
+            Global.changeProfileSection(7)
```

Comment:
This kinda feels weird. Why does the global scope have to know how to change profile sections?
Imagine a scenario where we're able lazy load modules, then the profile module may not even be instantiated yet, still there'd be an API to change profile sections.

I don't know from the top of my head what the function does, but judging by what it's called it feels like this wants to live in the profile module only.

Reply:
Hey Alex , I have a suggestion for this can we not have ENUMS defined for each of the sections in profile as well and have one single API.

Ex: appMain.changeAppSection(Constants.profile, Constants.ProfileSection.Browser) and in this way this can accommodate different things in the future for different modules? or just accept a second "params" in the same function.

Reply:
hey @Khushboo-dev-cpp updated, please have a look! I overlooked the places where the `changeProfileSection` was called and didn't realize that `changeAppSection` was also called before that and thus could be combined! thanks for pointing it out! :)